HW03/maksim_manzhai/maksim_manzhai_hw_03_t_02.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task2(logs)
```

Comment:
You have a complex solution, so you should refactor the code to make it readable:
Create 3 regular expressions to select DATETIME, IP, ADDRESS
Split the log you made on line 5
Select all lines that have a POST request
use an iterator on selected lines with POST request and collect datetime, ip and address using your regex
the last line in the iterator block will look like "#{datetime} FROM: #{ip} TO: #{address}"